src/components/footer/single-input-field/single-input-field.module.scss

```diff
@@ -29,3 +31,20 @@
     border-radius: 50%;
   }
 }
+
+.inputAnswer {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+  margin-bottom: -30px;
```

Comment:
not good, there is no good reason for doing negative px. Add 30 to another component instead